src/momento/config/middleware/models.py

```diff
@@ -0,0 +1,46 @@
+from typing import Dict
+
+import grpc
+from google.protobuf.message import Message
+
+CONNECTION_ID_KEY = "connectionID"
+
+
+class MiddlewareMessage:
```

Comment:
Everything here could be a dataclass. The sole field of this one would be `grpc_message`, then you could make property-like accessors for `message_length`, `constructor_name`. Accessing those in the code would be:

```python
m = MiddlewareMessage(message="asdf")
m.message_length
m.constructor_name
m.message
```